platforms/__common.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: can we add some suffix to the file name as I see two common.sh? it may be easier to maintain